### DISCUSSION TITLE
Jotai - Async basics

### DISCUSSION THREAD

**Discussion by BassWow** [pmndrs/jotai#460] [Show and tell]
Thought I would share some a codesandbox showing off your wonderful library using Async.

I have also shown how you can scope different components simply by using separate providers in the component tree. 

https://codesandbox.io/s/jotai-async-r2tk4

**Comment by dai-shi** [comment]
Nice!
Providers can be nested too, just in case.
Async write atom is not yet very common in the community, and something we will see how it goes.
You might want to try async read atom too.

**Reply by BassWow** [reply]
Thanks Dai-shi and once again an amazing piece of software my friend :) keep up the awesome work!
